spec/webvalve/manager_spec.rb

```diff
@@ -94,10 +97,17 @@
 
       it 'allowlists configured urls in webmock' do
         allow(WebMock).to receive(:disable_net_connect!)
-        results = [%r{\Ahttp://foo\.dev}, %r{\Ahttp://bar\.dev}]
+        results = [
+          %r{\Ahttp://foo\.dev#{url_suffix_pattern}},
```

Comment:
I don't know how annoying to devex these less legible WebMock URL patterns might be. Is this a problem?

Reply:
hm. i haven't found a way to hook into the webmock failure output and successfully rewrite the output so that we could swap in the * for the expanded pattern, if that's what you're getting at? 

i think that it could be surprising to folks if they encounter a big ugly regexp in the middle of the output but they could probably make the leap to understanding it's an expansion of the *

Reply:
oh but i guess you're saying that the suffixes get gnarly too now for all urls huh?

Reply:
No this actually goes on every URL now in order to defend against e.g. TLD expansion, which is called out in a test below.